cuesubmit/cuesubmit/Constants.py

```diff
@@ -38,7 +38,16 @@
 NUKE_RENDER_CMD = config.get('NUKE_RENDER_CMD', 'nuke')
 BLENDER_RENDER_CMD = config.get('BLENDER_RENDER_CMD', 'blender')
 FRAME_TOKEN = config.get('FRAME_TOKEN', '#IFRAME#')
-
+COMMAND_TOKENS = {'#ZFRAME#': 'Current frame with a padding of 4',
+                  '#IFRAME#': 'Current frame',
+                  '#FRAME_START#': 'First frame',
```

Comment:
I would say maybe `First frame of chunk` / `Last frame of chunk` to help clarify.

Reply:
Clarification applied.